config/import-control.xml

```diff
@@ -7,7 +7,18 @@
     <allow pkg="java.io"/>
     <allow pkg="java.util"/>
     <allow pkg="com.github.checkstyle.regression.data"/>
+    <allow pkg="com.github.checkstyle.regression.util"/>
+    <allow pkg="org.immutables"/>
```

Comment:
Keeping them as abstract classes seems to keep them requiring coverage, so lets stick with this.